components/form/fields/RatingField.tsx

```diff
@@ -0,0 +1,52 @@
+import {
+    Rating as MuiRatingField,
+    Typography,
+    FormHelperText,
+    FormControl
+} from '@mui/material'
+import { setColor } from '../utilities'
+
+type RatingFieldProps = {
+    name: string
+    field: { value: string; error: string }
+    onChange: any
+    id: string
+    label: string
+    submitError: boolean
+}
+
+const RatingField = ({
+    name,
+    field,
+    onChange,
+    id,
+    label,
+    submitError
+}: RatingFieldProps) => {
+    const parseValue = parseInt(field.value)
+    return (
+        <FormControl color={setColor(field)} error={!!field.error}>
```

Comment:
So another way to handle this...that i was made aware of in the extremely recent past (45 mins ago) is to do `Boolean(field.error)`. Let's go with that since Boolean is more legible, semantic, and a built in javascript function.